dts/arm64/nxp/nxp_mimx91.dtsi

```diff
@@ -141,6 +142,102 @@
 		clocks = <&ccm IMX_CCM_LPUART2_CLK 0x6c 24>;
 		status = "disabled";
 	};
+
+	lpi2c1: i2c@44340000 {
+		compatible = "nxp,lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x44340000 0x4000>;
+		interrupts = <GIC_SPI 13 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+		interrupt-parent = <&gic>;
+		clocks = <&ccm IMX_CCM_LPI2C1_CLK 0x70 6>;
+		status = "disabled";
+	};
+
+	lpi2c2: i2c@44350000 {
+		compatible = "nxp,lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x44350000 0x4000>;
+		interrupts = <GIC_SPI 14 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+		interrupt-parent = <&gic>;
+		clocks = <&ccm IMX_CCM_LPI2C2_CLK 0x70 8>;
+		status = "disabled";
+	};
+
+	lpi2c3: i2c@42530000 {
```

Comment:
Let's always order the nodes by unit address in ascending order just like Linux kernel,  it is recommended by Zephyr coding style rules:
https://docs.zephyrproject.org/latest/contribute/style/devicetree.html

Reply:
okay